release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -886,7 +886,38 @@ update could not be applied.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1711964[*BZ#1711964*])
 
 * The installer may have a `0` return code when hitting a FATAL event.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1712409[*BZ#1712409*])
+
+* After disabling service catalog, attempting to delete projects on the cluster
+can get stuck in a "Terminating" state. This is caused by the service catalog
+API service not deleting itself properly, and the deletion hangs. Service
+catalog does not properly clean up service instance resources that have a
+`kubernetes-incubator/service-catalog` finalizer; as a result, the namespace
+controller cannot finish the namespace (project) deletion.
++
+A change was made to service catalog to prevent the hangs by no longer
+attempting to delete its API service after disabling service catalog. If you
+have any projects stuck in a "Terminating" state after deleting service catalog,
+cluster administrators can use the following workaround:
++
+--
+. If you want to retain existing services that were deployed using service
```

Comment:
@jmrodri Based on my understanding, these retain services cannot work(cannot connect the cluster API server) well after deleting Service Catalog. So, are there any scenarios that the uses need to retain those services?

Reply:
The services the Ansible Service Broker deploys would work just fine. If mediawiki was deployed,  you should still be able to access it even without the service catalog. this may not be true for template service broker services. But the workaround is a way to remove the service catalog and brokers without having it also remove services that you may still want to use.

Reply:
I think in reality, most customers that would be removing service catalog and the brokers, would also redeploy their services using more modern tools such as operators. I don't think this is going to affect a huge number of customers.

Reply:
@jmrodri Yes, you're right. Agreed, thank you for your explanation!